typescript/packages/subsurface-viewer/src/SubsurfaceViewer.stories.tsx

```diff
@@ -445,3 +481,64 @@ ViewStateSynchronization.argTypes = {
         control: "check",
     },
 };
+
+export const IsLoadedCallback = (args: SubsurfaceViewerProps) => {
+    const [layers, setLayers] = React.useState([defaultWellsLayer] as [
+        WellsLayer,
+        MapLayer?
+    ]);
+    const [label, setLabel] = React.useState("");
+
+    const handleChange = () => {
+        if (layers.length === 1) {
+            setLayers([defaultWellsLayer, meshMapLayerBig]);
+        } else {
+            setLayers([defaultWellsLayer]);
+        }
+    };
+
+    const props = {
+        ...args,
+        isLoadedCallback: (isLoaded: boolean) => {
+            console.log("isLoadedCallback", isLoaded);
+            setLabel(isLoaded ? "LOADED" : "NOT LOADED");
+            return;
+        },
+        layers,
+    };
+
+    return (
+        <Root>
+            <div className={classes.main}>
+                <SubsurfaceViewer {...props} />
+            </div>
+            <button onClick={handleChange}> Add/remove layer </button>
```

Comment:
Maybe you could use a switch or toggle instead, saying "Show map layer"?

Reply:
I am not quite sure how to make a checkbox but I changed the text dynamically to say either "Add layer" or "Remove layer" depending.

Reply:
Used mui switch instead